frontend/lib/components/TextEditor/TextEditor.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+import Focus from "@tiptap/extension-focus";
+import { Link } from "@tiptap/extension-link";
+import { EditorContent, useEditor } from "@tiptap/react";
+import { StarterKit } from "@tiptap/starter-kit";
+
+import styles from "./TextEditor.module.scss";
+import { Toolbar } from "./components/Toolbar/Toolbar";
+
+import { SearchBar } from "../ui/SearchBar/SearchBar";
+
+const defaultContent = `
```

Comment:
Maybe you can remove the default content here, as we saif at the daily

Just add a New Note header, and a quick tutorial about brains usage with markdown